### PR TITLE
Help User Enter Model Dir

### PR DESCRIPTION
**Pull request by hyunsik-yoon**
This adds UI to help user to enter model directory.

parent: #330 

ONE-vscode-DCO-1.0-Signed-off-by: Hyun Sik Yoon <hyunsik.yoon.1024@gmail.com>

### CODE REVIEW

**Comment by hyunsik-yoon** [comment]
UI Flow

**1) Open Folder button**
image

**2) Dialog to enter dir**
image

**3) set setting and show message (temporarily)**
image

**Review by YongseopKim — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -52,6 +52,11 @@
     ],
     "commands": [
       {
+        "command": "onevscode.fileTreeView-open",
+        "title": "Set model directory",
+        "category": "ONE",
+        "_comment": "Call this command to set the model directory of local machine."
+      },      {
```
Comment (jyoungyun) [inline package.json:59]: You need a new line.

```suggestion
      },
      { 
```

Comment (hyunsik-yoon) [reply package.json]: Fixed. Thanks.

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by YongseopKim — APPROVED** [review]

**Pull request merged**